linter_exclusions.yml

```diff
@@ -3942,3 +3942,8 @@ webapp update:
     skip_dns_registration:
       rule_exclusions:
       - option_length_too_long
+synapse workspace create:
+  parameters:
+    managed_resource_group_name:
+      rule_exclusions:
+      - option_length_too_long
```

Comment:
Can you add a short option for this parameter instead of adding linter exclusion?